sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Azure.Verticals.AgriFood.Farming.csproj

```diff
@@ -18,7 +19,6 @@
   <!-- Shared source from Azure.Core -->
   <ItemGroup>
     <Compile Include="$(AzureCoreSharedSources)ArrayBufferWriter.cs" LinkBase="Shared" />
-    <Compile Include="$(AzureCoreSharedSources)AzureKeyCredentialPolicy.cs" LinkBase="Shared" />
```

Comment:
Reiterating @christothes's question here - why is this preferred over explicitly including relevant files?  

I think the approach of defining new MSBuild properties has a lot of problems, and I would like us to move way from it.  As part of the work I was doing on https://github.com/Azure/azure-sdk-for-net/issues/28369, I was hoping to remove `<IncludeOperationsSharedSource>` eventually.  I would prefer not to be adding new properties, as this introduces technical debt that we want to eliminate eventually, so it is solving a short term problem but creating more work for us in the future.